app/src/main/java/org/oppia/app/help/faq/FAQListFragmentPresenter.kt

```diff
@@ -29,7 +29,11 @@ class FAQListFragmentPresenter @Inject constructor(
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
     val viewModel = getFAQListViewModel()
 
-    binding = FaqListFragmentBinding.inflate(inflater, container, /* attachToRoot = */ false)
+    binding = FaqListFragmentBinding.inflate(
+      inflater,
+      container,
+      /* attachToRoot = */ false
```

Comment:
`/* attachToRoot= */`

Reply:
will it look good? as in common standard wherever we assign there are spaces on both side of `=`. same in comment we follow

Reply:
No you can check in other classes. Lets keep it consistent.

Reply:
ok, I am updating

Reply:
Thanks.

Reply:
Done